viewing_party/party.py

```diff
@@ -1,23 +1,146 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        movie_dict = {"title": title, "genre": genre, "rating": rating}
+        return movie_dict
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    '''
+    Takes two arguments, user_data and movie, where user_data 
+    is a dictionary that stores all of the movies the user has watched 
+    and movie is a dictionary which contains the information for a movie
+    
+    Adds the value of movie to the "watched" list inside of user_data
+
+    Returns user_data
+    '''

```

Comment:
Yep, this is actually exactly how you should write a docstring! And it's thorough and well written too! I could dive down into your other docstrings but most of them already seem pretty good, and we're a bit more concerned with code writing right now than documentation writing.

I do encourage you to continue to practice the skill, it's definitely underrepresented among programmers in the industry!